scripts/metadata_table.py

```diff
@@ -1266,6 +1279,36 @@ def is_scalar_reference(test_val):
         """Return True iff <test_val> refers to a Fortran scalar."""
         return check_fortran_ref(test_val, None, False) is not None
 
+    def to_html(self, outdir, props):
+        """Write html file for metadata section and return filename.
+        Skip metadata sections without variables"""
+        if not self.__variables.variable_list():
+            return None
+        # Write table header
+        header = "<tr>"
+        for prop in props:
+            header += "<th>{}</th>".format(prop)
```

Comment:
I have been converting to f strings, this seems like a good place.

Reply:
I have adopted f strings in this PR, thanks for the suggestion.